lms/validation/authentication/_bearer_token.py

```diff
@@ -60,6 +60,10 @@ class BearerTokenSchema(PyramidRequestSchema):
     user_id = marshmallow.fields.Str(required=True)
     oauth_consumer_key = marshmallow.fields.Str(required=True)
     roles = marshmallow.fields.Str(required=True)
+    tool_consumer_instance_guid = marshmallow.fields.Str(required=True)
+    given_name = marshmallow.fields.Str(required=True)
+    family_name = marshmallow.fields.Str(required=True)
+    full_name = marshmallow.fields.Str(required=True)
```

Comment:
Adding these to `BearerTokenSchema` means that:

1. When the backend serializes `request.lti_user` into a bearer token to go in the JS config, that seralized bearer token now contains these additional fields

2. When the frontend uses that bearer token to authenticate an API request and the backend deserializes the bearer token into a `request.lti_user` object, that `LTIUser` object will now have the new fields